kube_hunter/modules/hunting/etcd.py

```diff
@@ -88,9 +88,7 @@ def db_keys_write_access(self):
         data = {"value": "remotely written data"}
         try:
             r = requests.post(
-                f"{self.protocol}://{self.event.host}:{ETCD_PORT}/v2/keys/message",
-                data=data,
-                timeout=config.network_timeout,
+                f"https://{self.event.host}:{ETCD_PORT}/v2/keys/message", data=data, timeout=config.network_timeout,
```

Comment:
Should this be `{self.event.protocol}`? I didn't check but I'm wondering if OpenPortEvent populates this. (In practice I would imagine any self-respecting deployment will use https anyway, so it's not a big deal, but just wondering about it)

Reply:
@lizrice 
Open port event does not populate it... but I changed it to save the "https" in the event, in any case if we want to use it in other event later on.